js/src/components/free-listings/configure-product-listings/shipping-time/shipping-time-setup/getCountriesTimeArray.js

```diff
@@ -26,10 +34,12 @@
  * //     {
  * //         countries: ['US', 'AU'],
  * //         time: 10
+ *  //		  maxTime: 20,
  * //     },
  * //     {
  * //         countries: ['CN'],
  * //         time: 15
+ * //		  maxTime: 22,
```

Comment:
Avoid mixed indentation chars.

```suggestion
 * //         maxTime: 22,
```